views/default/input/date.js

```diff
@@ -0,0 +1,62 @@
+define(function (require) {
+
+	var elgg = require('elgg');
+	var $ = require('jquery');
+	require('jquery-ui');
+
+	if (elgg.get_language() !== 'en') {
+		// require language module if necessary
+		require('jquery-ui/i18n/datepicker-' + elgg.get_language() + '.min');
```

Comment:
I thought `define(function(require) {});` pattern is equivalent to async `define([], function() {})`